charts/drone/templates/deployment-server.yaml

```diff
@@ -55,16 +55,20 @@ spec:
       initContainers: {{- include "extraRootCA.init" (dict "image" $image "rootCA" .Values.extraRootCA) | nindent 6 }}
       {{- if .Values.otomiApiUsesOtomiGitea }}
         - name: waitforgitea
-          image: alpine/git
-          script: |
-            {{ if (merge .Values dict) | dig "env" "GIT_SSL_NO_VERIFY" nil }}NOSSL=' -c http.sslVerify=false'{{ end }}
-            GITEA_URL=https://$DRONE_GIT_USERNAME:$DRONE_GIT_PASSWORD@gitea.{{ .Values.clusterDomainSuffix }}/otomi/values
-            echo "Waiting until gitea is accessible at gitea.{{ .Values.clusterDomainSuffix }}/otomi/values"
-            until $(git $NOSSL ls-remote $GITEA_URL); do 
-              printf '.'
-              sleep 5
-            done
-            echo READY!
+          image: alpine/git:v2.30.2
+          # FIXME: why do we need to wait for gitea?
```

Comment:
Because gitea needs to generate a secret for drone. Drone sets this on startup. So cannot start until gitea is started & secret has been created